homeassistant/components/rflink.py

```diff
@@ -403,6 +416,42 @@ def async_turn_off(self, **kwargs):
         return self._async_handle_command("turn_off")
 
 
+class CoverableRflinkDevice(RflinkCommand):
```

Comment:
I think this was lost in a previous pullrequest, but there is no need to have this code in the main rflink module as it is only used by the cover. You can just put all this code in the cover.py module, like is done with light classes for example: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/light/rflink.py#L262

Reply:
moved the code to cover/rflink.py, hopefully it's correct now.

Reply:
This class can be removed here completely as right now it does nothing :)